assets/dev/js/editor/document/elements/commands/create.js

```diff
@@ -48,7 +48,13 @@ export class Create extends $e.modules.editor.document.CommandHistoryBase {
 		containers.forEach( ( container ) => {
 			container = container.lookup();
 
-			const createdContainer = container.view.addElement( model, options ).getContainer();
+			const createdElement = container.view.addElement( model, options );
+
+			if ( ! createdElement ) {
```

Comment:
I also came across the error here when I was playing with it.
My questions are - does this fix the problem or just handle the error? Isn't an element supposed to be created?

Reply:
@udidol From what I can see this creates the content container - which we don't need.

Reply:
@udidol I saw that error came from "creating" a new item, not duplicating. I removed this code as we have a separate task for that, and this code only handled the error. the issue was the `nested-repeater-create-container.js` should not be calling this file in the first place for Link Only items.